### PR TITLE
[#9508] Improve UI for Instructor Course Details

### LINKED ISSUE #9508: Instructor: Course Details: Improve UI
**Environment**:
`master` branch

**Steps to reproduce**
1. Go to `web/instructor/courses/details?courseid=<valid course id with students>`
2. Observe screenshot posted:
image

**Things to fix**:
1. Remove the extra `Students` div
2. Align the `student-list-component` with the card body.

### PR DESCRIPTION
**Pull request by ritisha2000**
Fixes #9508  

I found the file instructor-course-details-page.component.html in pages-instructor\instructor-course-details-page\instructor-course-details-page.component.html and removed the student header. I also changed the card-body class p-3 to p-0 align it to the card body above.

### CODE REVIEW

**Comment by ritisha2000** [comment]
CoureDetails
I checked what the error was and it is failing a test because the jest snapshot is not updated and it is comparing it to the old snapshot which contains the student div. I am working on changing that but here's what the edits look like.
Thank you for your feedback.

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "@types/node": "~8.9.4",
     "codelyzer": "~4.5.0",
     "jest": "^23.6.0",
+    "jest-cli": "^24.5.0",
```
Comment (tanhengyeow) [inline package.json]: Just checking, is this change needed for this patch to work? If not let's remove it.

File: src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html [hunk 2]
```diff
@@ -354,7 +354,7 @@
               <div class="form-group">
                 <div class="align-center">
                   <button class="btn btn-primary" tabindex="6" id="btn-save-{{ i }}" style="display:none;" type="submit">
-                    Save changes
+                    Save Changes
```
Comment (tanhengyeow) [inline src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html]: Same here.

File: src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.html [hunk 3]
```diff
@@ -44,7 +44,7 @@ <h1>Course Details</h1>
             <ng-template #showRole >
               <p>
                 {{instructor.role}} {{instructor.name}} ({{instructor.email}})
-              </p>
+              </p>no
```
Comment (tanhengyeow) [inline src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.html]: This seems like an unrelated change.

**Comment by tanhengyeow** [comment]
Also, although not strictly required, it would be nice to tidy up your PR description.

Here are some examples below:

1. We always put `Fixes #....`  at the top of the description.

2. This chunk of info seems irrelevant:
```
…artinoamigo/teammates into 9508_Course_Details_Improve_UI

[#9508] Instructor: Course Details: Improve UI
```

3. It's good to use backticks to differentiate file names and file paths e.g. `instructor-course-details-page.component.html` and `pages-instructor\instructor-course-details-page\instructor-course-details-page.component.html` (just the relative path would suffice)

File: package.json (hunk 1 above)
Comment (ritisha2000) [reply package.json]: This is a change made by jest when updating the snapShot so it doesn't give an error during checks.

**Comment by ritisha2000** [comment]
There were two unnecessary changes so I removed them and changed the description. Thank you for your help.

File: package.json (hunk 1 above)
Comment (wkurniawan07) [reply package.json]: @ritisha2000 I don't believe so. You should not need `jest-cli` in the first place. How are you running the tests?

Comment (ritisha2000) [reply package.json]: I assumed it was necessary since it wasn't added by me. I ran the "npm test" command it doesn't seem to show any failing tests without the line so I will remove it.

**Comment by tanhengyeow** [comment]
@ritisha2000 Close to merging :)

> You shouldn't need to update src/test/resources/pages/* files. Those are legacy files that we are trying to deprecate.

Refer to this comment again.

File: src/test/resources/pages/instructorCourseDetailsWithSections.html [hunk 4]
```diff
@@ -160,168 +160,7 @@ <h1>
...
-          <a class="course-student-delete-link btn btn-default btn-xs margin-bottom-7px" data-course-id="CCDetailsUiT.CS2103" data-original-title="Delete the student and the corresponding submissions from the course" data-placement="top" data-student-name="Benny Charles" data-toggle="tooltip" href="/page/instructorCourseStudentDelete?courseid=CCDetailsUiT.CS2103&studentemail=benny.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&token=${sessionToken}" title="">
-            Delete
-          </a>
-          <a class="btn btn-default btn-xs margin-bottom-7px" data-original-title="View all data about this student" data-placement="top" data-toggle="tooltip" href="/page/instructorStudentRecordsPage?courseid=CCDetailsUiT.CS2103&studentemail=benny.tmms%40gmail.tmt&user=CCDetailsUiT.instr2&token=${sessionToken}" rel="noopener noreferrer" target="_blank" title="">
-            All Records
-          </a>
-        </td>
-      </tr>
-    </tbody>
-  </table>
```
Comment (tanhengyeow) [inline src/test/resources/pages/instructorCourseDetailsWithSections.html]: @ritisha2000 You missed out this comment :)

> You shouldn't need to update src/test/resources/pages/* files. Those are legacy files that we are trying to deprecate.

Comment (ritisha2000) [reply src/test/resources/pages/instructorCourseDetailsWithSections.html]: Sorry about that. I have taken out the changes I made. Thank you for reminding me.

**Comment by ritisha2000** [comment]
Thank you for all your help
